ArduPlane/Log.cpp

```diff
@@ -306,12 +308,14 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: RdrOut: scaled output rudder
 // @Field: ThD: demanded speed-height-controller throttle
 // @Field: As: airspeed estimate (or measurement if airspeed sensor healthy and ARSPD_USE>0)
+// @Field: AsStat: airspeed status (unhealthy or source of estimate)
```

Comment:
What is the value for unhealthy?

Reply:
unhealthy only applies to the sensor not the estimate....the estimate can be a valid source or NO NEW ESTIMATE...ie its the same as the last valid estimate

Reply:
The enum is here: https://github.com/ArduPilot/ardupilot/pull/25080/files#diff-11827c8755e1c0219f3765d61dc55865a81b9d1a9e72daf33c4b4cb6e390580bR833-R840

Reply:
Yes I see that, but I'm thinking about how to read the values when viewing the log. Are you just going to see some numbers (-1, 1,2,3)... could the comment message not say what the values mean?
[The comment message is what you get as a mouse over in the log viewer I think]

Reply:
not our pattern...its followed by a FieldValueENUM that lists the values in the wiki automatically...not opposed if Tridge and Peter approve....

Reply:
This is great - just one thought. Perhaps it's a bit late, but this abbreviation discussion just made me think of it - how about AsSrc (source) rather than AsStat?

Reply:
no new estimate does not tell you the source...